package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
+    "prop-types": "^15.7.2",
```

Comment:
# [Optional]
Let's try to add `big.js` as it was stated in milestone 1
![image](https://user-images.githubusercontent.com/31897434/118617407-83533300-b7ba-11eb-96ec-73abca8e677f.png)